changelog.md

```diff
@@ -1,3 +1,9 @@
+# NEXT
+- [FIXED] Fix findOrCreate regression trying to add a transaction even if there is none
+
+# 3.4.1
+- [FIXED] Fix belongs-to-many ambigious id when through model has id
```

Comment:
For `createAssociation`, that changelog entry should probably have been reviewed before merge, looks its's just been copyied from a commit message without checking thoroughly ;p